Update receiving-grants.md

giacastello opened the pull request:
Adding maximum period for pausing a vesting contract, according to this proposal: https://decentraland.org/governance/proposal/?id=d20c8871-b7b2-4a87-b692-b84ef3331c60

nearnshaw commented:
Thanks @Converg3nce for taking the time to review all of these documents with such a keen eye <3

I have merged all of your PRs, and the changes will be live shortly

giacastello committed:
Update receiving-grants.md

Adding maximum period for pausing a vesting contract, according to this proposal: https://decentraland.org/governance/proposal/?id=d20c8871-b7b2-4a87-b692-b84ef3331c60

Signed-off-by: Gia ░░░░░░░░░░░░▒▒▒▒▒▒▒▒▒▒▒▒ Castello <giacastello@gmail.com>

nearnshaw committed:
Merge branch 'main' into patch-10
